src/modules/kvs/cache.h

```diff
@@ -89,6 +89,13 @@ int cache_entry_set_errnum_on_notdirty (struct cache_entry *entry, int errnum);
 int cache_entry_wait_notdirty (struct cache_entry *entry, wait_t *wait);
 int cache_entry_wait_valid (struct cache_entry *entry, wait_t *wait);
 
+/* Get the blobref this entry is stored in the cache with.  Generally
+ * speaking, this should not be used except in error handling
```

Comment:
Why restrict its use?  Isn't it valid for any cache entry?

Reply:
Yeah, this comment isn't great.  The primary point was that the caller should use the result from ```flux_content_store_get()``` to double check that the reference returned from the content store matches the cache reference.  And if there is an error, then use this.  It's probably just best to remove this comment.